docs/connector-development/connector-builder-ui/partitioning.md

```diff
@@ -0,0 +1,116 @@
+* The "Inject partition value into outgoing HTTP request" option allows to configure how to add the current partition value to the requests
+
+#### Example
+
+To enable static partition routing for the [SurveySparrow API](https://developers.surveysparrow.com/rest-apis/response#getV3Responses) responses, the list partition router needs to be configured as following:
+* "Partition values" are set to the list of survey ids to fetch
+* "Current partition value identifier" is set to `survey` (this is not used for this example)
+* "Inject partition value into outgoing HTTP request" is set to `request_parameter` for the field name `survey_id`
+
+When partition values were set to `123`, `456` and `789`, the following requests will be executed:
+```
+curl -X GET https://api.surveysparrow.com/v3/responses?survey_id=123
+curl -X GET https://api.surveysparrow.com/v3/responses?survey_id=456
+curl -X GET https://api.surveysparrow.com/v3/responses?survey_id=789
+```
+
+To enable static partitions for the [Woocommerce API](https://woocommerce.github.io/woocommerce-rest-api-docs/#order-notes) order notes, the configuration would look like this:
+* "Partition values" are set to the list of order ids to fetch
+* "Current partition value identifier" is set to `order`
+* "Inject partition value into outgoing HTTP request" is disabled, because the order id needs to be injected into the path
+* In the general section of the stream configuration, the "Path URL" is set to `/orders/{{ stream_slice.order }}/notes`
+
+
+When partition values were set to `123`, `456` and `789`, the following requests will be executed:
+```
+curl -X GET https://example.com/wp-json/wc/v3/orders/123/notes
+curl -X GET https://example.com/wp-json/wc/v3/orders/456/notes
+curl -X GET https://example.com/wp-json/wc/v3/orders/789/notes
+```
+
+### Substream partition router
+
+To fetch the list of partitions (in this example surveys or orders) from the API itself, the "Substream" partition router has to be used. It allows you to select another stream of the same connector to serve as the source for partitions to fetch. Each record of the parent stream is used as a partition for the current stream.
+
+The following fields have to be configured to use the substream partition router:
+* The "Parent key" is the property on the parent stream record that should become the partition value (in most cases this is some form of id)
+* The "Current partition value identifier" can be freely choosen and is the identifier of the variable holding the current partition value. It can for example be used in the path of the stream using the `{{ stream_slice.<identifier> }}` interpolation placeholder.
+* The "Parent stream" defines the records of which stream should be used as partitions
+
+#### Example
```

Comment:
well done on the explanation for substream routers. I always feel like this one can get confusing given the added complexity of multiple streams and identifiers